packages/testdata/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@ethereumjs/testdata",
+  "version": "1.0.0",
+  "description": "Common test data for EthereumJS packages",
+  "type": "module",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    }
+  },
+  "files": ["dist", "src"],
+  "scripts": {
+    "build": "../../config/cli/ts-build.sh",
```

Comment:
Do we actually need a build step here?

Reply:
My underestanding is that it's best to do it to enable tests to be run with nodejs (otherwise users would need to use tsx/ts-node)

Reply:
I deleted the dist folder from `testdata` and ran all the tests locally and they pass just fine.  This is what I expected since our code is intended to use typescript sources and vitest uses a transpiler internally to handle typescript code.  Since this package is for local testing only and not intended to be published, I'd advocate to skip the build step entirely so as to not add anything else to the CI (even though this admittedly won't add much)

Reply:
Ah! Nice, thanks for actually trying this out. Will remove the build step and add the necessary --if-present flag so that the root postinstall doesn't fail on that package.

Reply:
Done! Feel free to re-approve! :)